### PR TITLE
docs: install crdb chart separately

### PR DESCRIPTION
**Pull request by eliobischof**
depends on https://github.com/zitadel/zitadel-charts/pull/48

### CODE REVIEW

File: docs/docs/guides/deploy/kubernetes.mdx [hunk 1]
```diff
@@ -12,23 +12,29 @@ For running an easily testable, insecure, non-HA ZITADEL instance, run the follo
...
+  --set single-node=true \
+  --set statefulset.replicas=1
+
+# ZITADEL
 helm install --namespace zitadel --create-namespace my-zitadel zitadel/zitadel \
   --set zitadel.masterkey="MasterkeyNeedsToHave32Characters" \
   --set zitadel.configmapConfig.ExternalSecure=false \
   --set zitadel.configmapConfig.TLS.Enabled=false \
   --set zitadel.secretConfig.Database.cockroach.User.Password="a-zitadel-db-user-password" \
   --set replicaCount=1 \
```
Comment (thesephirot) [inline docs/docs/guides/deploy/kubernetes.mdx:37]: this should be deleted, otherwise the command won't execute

Comment (thesephirot) [reply docs/docs/guides/deploy/kubernetes.mdx:37]: marked the wrong lines, sry. I mean the last char at 29

Comment (eliobischof) [reply docs/docs/guides/deploy/kubernetes.mdx]: thx

**Comment by fforootd** [comment]
Since the upstream since to be fixed, I guess this can be merged soon, right?

**Comment by thesephirot** [comment]
Right, it was testen with this docs

**Review by thesephirot — APPROVED** [review]
Tested

**Pull request merged**